client/src/use/useVuetify.ts

```diff
@@ -0,0 +1,8 @@
+import { getCurrentInstance } from 'vue';
+
+const useVuetify = () => {
+  const vm = getCurrentInstance();
+  return vm?.proxy?.$vuetify || undefined;
```

Comment:
Is it reasonable at runtime for this function to return undefined, or does that mean something unexpected has gone wrong?

Reply:
Good call, If this is undefined it should error out.  I'll update to throw an error.